File: Library2/Models/Borrowed_Book.cs
namespace Library2.Models
{
    public class Borrowed_Book
    {
        public int Id { get; set; }
       
        public int client_id { get; set; }
        public string User_id { get; set; }

        public int Book_id { get; set; }
        public string Book_Title { get; set; }
        public string Start_Date { get; set; } = DateTime.Now.Date.ToString();
        public string End_Date { get; set; }
    }
}

File: Library2/wwwroot/js/Borrowed_Book.js
$(document).ready(() => {

    $.ajax({

        url: "/Borrowed_Book/fetchData",
        success: function (data) {
            console.log(data);
            for (var i = 0; i < data.length; i++) {
               // $('tbody').append(' <tr> <td>'+data[i].user_id+'</td>  <td>$data[i].book_id</td> <td>${data[i].book_Title}</td> <td>${data[i].start_Date}</td> <td>${data[i].end_Date}</td> </tr>');

            }
        },
        error: function (er) {
            console.log(er);
        }

    });

});